component.json

```diff
@@ -141,6 +141,70 @@
         }
       },
       "dynamicMetadata": true
+    },
+    "upsertSpreadsheetRow": {
+      "main": "./lib/actions/upsertSpreadsheetRow.js",
+      "title": "Create/Upsert/Update Spreadsheet Row",
+      "help": {
+        "description": "Action appends new row/column to spreadsheet or update it",
+        "link": "/components/gspreadsheet/actions#upsert-spreadsheet-row"
+      },
+      "fields": {
+        "spreadsheetId": {
+          "viewClass": "SelectView",
+          "prompt": "Select your spreadsheet",
+          "label": "Spreadsheet",
+          "required": true,
+          "model": "listSpreadsheets"
+        },
+        "worksheetId": {
+          "viewClass": "SelectView",
+          "prompt": "Select your worksheet (spreadsheet required)",
```

Comment:
I think unnecessary part `(spreadsheet required)`, it will be gray anyway if `Spreadsheet` empty